# HG changeset patch
# User Mike Miller <mtmiller@ieee.org>
# Date 1376394760 14400
#      Tue Aug 13 07:52:40 2013 -0400
# Node ID 2736bc7bf8d993070141512c3ac2396f597b2665
# Parent  2f1729cae08fa31c0bc16d35907666ee9c53f783
Fix syntax error typos in spectral_adf and spectral_xdf (bug #39766)

* spectral_adf.m, spectral_xdf.m: Fix syntax error typos.

diff --git a/scripts/signal/spectral_adf.m b/scripts/signal/spectral_adf.m
--- a/scripts/signal/spectral_adf.m
+++ b/scripts/signal/spectral_adf.m
@@ -42,17 +42,17 @@ function retval = spectral_adf (c, win, 
 
   if (nargin < 3)
     b = 1 / ceil (sqrt (cr));
   endif
 
   if (nargin == 1)
     w = triangle_lw (cr, b);
   else
-    win = str2func ([win "_lw")];
+    win = str2func ([win "_lw"]);
     w = feval (win, cr, b);
   endif
 
   c = c .* w;
 
   retval = 2 * real (fft (c)) - c(1);
   retval = [(zeros (cr, 1)), retval];
   retval(:, 1) = (0 : cr-1)' / cr;
diff --git a/scripts/signal/spectral_xdf.m b/scripts/signal/spectral_xdf.m
--- a/scripts/signal/spectral_xdf.m
+++ b/scripts/signal/spectral_xdf.m
@@ -41,17 +41,17 @@ function retval = spectral_xdf (x, win, 
 
   if (nargin < 3)
     b = 1 / ceil (sqrt (xr));
   endif
 
   if (nargin == 1)
     w = triangle_sw (xr, b);
   else
-    win = str2func ([win "_sw")];
+    win = str2func ([win "_sw"]);
     w = feval (win, xr, b);
   endif
 
   x = x - sum (x) / xr;
 
   retval = (abs (fft (x)) / xr).^2;
   retval = real (ifft (fft (retval) .* fft (w)));
 
